### DISCUSSION TITLE
gpg signature

### DISCUSSION THREAD

**Discussion by cdaaawg** [ventoy/Ventoy#2307] [General]
It would be helpful if the downloadable files had a gpg signature available for checking authenticity of the file.

**Comment by findsaplac** [comment]
I would support the issue. it seems to me that this is security-related software & might be valuable to have it provided.
maybe youve got good reasons, not to give it, but can imagine, that you've just not come about prioritizing it?
thanks if it is possible, that you shift it up the agenda or give a short note how come that you consider it not that much relevant (might be above my knowledge:).
I'd appreciate & would like to say thank you for the effort you provide Ventoy:) I really am a fan, kind regards!
